doc/managing-wallet.md

```diff
@@ -0,0 +1,85 @@
+# Managing the Wallet
+
+## 1. Backing Up and Restoring The Wallet
+
+### 1.1 Creating the Wallet
+
+Since version 0.21, Bitcoin Core no longer has a default wallet.
+Wallets can be created with the RPC command `createwallet`. More information about this command can be found running `bitcoin-cli help createwallet`.
+
+The following command, for example, creates a descriptor wallet:
+
+`$ bitcoin-cli -named createwallet wallet_name="wallet-01" descriptors=true`
+
+The `descriptors` parameter can be omitted if the intention is to create a legacy wallet.
+
+For now, the default type is the legacy wallet, but that should change in the near future.
+
+Wallets are created in the `wallets` folder of the data directory. The default folder varies by operating systems, as shown below. The user can also change the default by using the `-datadir` or `-walletdir` initialization parameters.
+
+| Operating System | Default wallet directory                                    |
+| -----------------|:------------------------------------------------------------|
+| Linux            | `home/<user>/.bitcoin/wallets`                              |
+| Windows          | `C:\Users\<user>\AppData\Roaming\Bitcoin\wallets`           |
+| macOS            | `/Users/<user>/Library/Application Support/Bitcoin/wallets` |
+
+### 1.2 Encrypting the Wallet
+
+The `wallet.dat` file is not encrypted by default and is, therefore, vulnerable if an attacker gains access to the device where the wallet or the backups are stored.
+
+The wallet may be encrypted with the following command:
+
+`$ bitcoin-cli -rpcwallet="wallet-01" encryptwallet "passphrase"`
+
+The `encryptwallet` command is used only when the wallet has not been encrypted yet. Otherwise, the `walletpassphrasechange` command should be used.
+
+`$ bitcoin-cli -rpcwallet="wallet-01" walletpassphrasechange "oldpassphrase" "newpassphrase"`
+
+The argument passed to `-rpcwallet` is the name of the wallet to be encrypted.
+
+The term "encrypt the wallet" used here is not very accurate. This command encrypts only the private key. All other wallet information, such as transactions, is still visible.
+
+The wallet can also be encrypted in the `createwallet` command via the `passphrase` argument:
+
+`$ bitcoin-cli -named createwallet wallet_name="wallet-01" descriptors=true passphrase="passphrase"`
+
+Note that if the passphrase is lost, all the coins in the wallet will also be lost forever.
+
+### 1.3 Backing Up the Wallet
+
+Wallets can be safely copied to another destination. This backup file should be stored on offline devices only, such as on a USB drive or an external hard drive.
```

Comment:
```suggestion
Wallets can be copied to another destination. This backup file should be stored on offline devices only, such as on a USB drive or an external hard drive.
```

It may be clearer for this paragraph and the `backupwallet` one to be together so the user doesn't interpret "copy" differently.